lib/koans/03_numbers.ex

```diff
@@ -109,6 +109,10 @@ defmodule Numbers do
     assert Float.round(-5.5674, 3) === ___
   end
 
+  koan "But beware of floating dragons" do
+    assert Float.round(2.675, 2) == ___
```

Comment:
There is a test suite for the koans. Would you mind adding a corresponding test for this? (My guess is that the CI run that I just started will fail and point you in the right direction)

Reply:
oof, actually it looks like CI is broken atm 😬 ...

Reply:
You should be able to run `mix test` on the app locally.

Reply:
> 49 tests, 0 failures

Thanks !  It didn't even occur to me that it was testable.  I see the Blanks now.  Pretty crafty !   :D

Reply:
Looks good! Now my turn to fix the CI 😆